src/lib/y2network/interface.rb

```diff
@@ -132,5 +132,10 @@ def update_udev_rule
           Y2Network::UdevRule.new_bus_id_based_rename(name, hardware.busid, hardware.dev_port)
       end
     end
+
+    # @return [Boolean] true if the interface is hotplug
+    def hotplug?
+      ["usb", "pcmci"].include?(hardware["hotplug"])
```

Comment:
`pcmci` or `pcmcia`? I know, the original one was `pcmci`. Just curious.

Reply:
good question.  "pcmci" has been here longer than I work with the code. I've checked source of hwinfo / libhd. And it seems that PCMCIA is correct (as expected anyway)

See https://github.com/openSUSE/hwinfo/blob/4264ba23bc432193d0e1a522909e299cfb35b468/src/hd/hdp.c#L369

Reply:
It's `pcmcia`. And I really wonder if we should care about it at all or just remove that stuff.

Reply:
thanks ... so it obviously was wrong for about 10 years

Reply:
Which nicely illustrates why we might remove it.

Reply:
So it remained unused for about 10 years :smile: Actually, we removed another PCMCIA-related feature (in the autoinstallation rules) back in 2016. It never worked for SLE 12. So I agree with @wfeldt here.